platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/Platform.java

```diff
@@ -45,6 +46,14 @@ public interface Platform {
     @NonNull
     NotificationEngine getNotificationEngine();
 
+    /**
+     * Get the Roster
+     *
+     * @return the roster
+     */
+    @NonNull
+    Roster getRoster();
```

Comment:
I highly advise against adding a method like this to the `Platform` interface. It will be convenient now, but hard to make thread safe when DAB takes effect. I know there's an existing `getAddressBook()` method, but that method is one that should be phased out, not carried forward. The core issue is that pretty much anything that cares about the roster needs to be aware of how it changes and carefully control access at round boundaries. Much better to allow the application to subscribe to roster updates if they want access outside of the handle transaction thread, and to read it from the state while on the handle transaction thread.

Reply:
The API changes that would be required to remove this method are beyond the scope of this PR. Right now the goal is just to get rid of address book usage. 

Fully dynamic address book isn't actively being worked right now, IMO this is a decision that needs to be made during that effort.

Reply:
I agree with @litt3,  the platform refactor for TSS is blocked if we can't get rid of `AddressBook` usage.   DAB will have to be significantly replanned anyway after TSS.    We should replace the `getAddressBook` with `getRoster` for now.